packages/common/src/core/HubItemEntity.ts

```diff
@@ -1,4 +1,5 @@
 import {
+  addItemResource,
```

Comment:
Is there a reason these imports are added here (e.g. `addItemResource`, `objectToJsonBlob`, and `stringToBlob`? I don't see them being used